bootstrap.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 
-OSIAM_VERSION=2.4
+OSIAM_VERSION=3.0
```

Comment:
I think we want a SNAPSHOT version. It just hasn't been updated after the last release :)